server/channels/app/authorization.go

```diff
@@ -21,7 +21,7 @@ func (a *App) SessionHasPermissionTo(session model.Session, permission *model.Pe
 	return a.RolesGrantPermission(session.GetUserRoles(), permission.Id)
 }
 
-func (a *App) SessionHasPermissionToAny(session model.Session, permissions []*model.Permission) bool {
+func (a *App) SessionHasPermissionToAny(c request.CTX, session model.Session, permissions []*model.Permission) bool {
```

Comment:
`request.CTX` already contains a Session. I wonder if we could simply drop the `session` argument.

@isacikgoz What do you think?

Reply:
@hanzei , Should I change this or wait for confirmation?